### ISSUE TITLE
Translation : Persian

### ISSUE THREAD

**Issue by ghaseminya** [jwasham/coding-interview-university#186]
Hi @jwasham 
I wana to translate this repo in Persian language(fa-ir).
Is this job started before?
Thanks.

**Comment by jwasham** [comment]
Not yet.
Thanks for starting it!
Make an issue and a pull request to track your changes and collaborate with others.

**Comment by EmpireWorld** [comment]
@ghaseminya Can i help you in this process?

**Comment by ghaseminya** [comment]
Hi @EmpireWorld 
Thanks for attention,
I started translation of Readme.md to README-fa.md in root folder
https://github.com/ghaseminya/google-interview-university/blob/master/README-fa.md
Is this job is true and i continue to it?

**Comment by EmpireWorld** [comment]
@ghaseminya I want to join the persian translation.

**Comment by ghaseminya** [comment]
@EmpireWorld i think you are from project team!
So coworking in translation good job
I start translation readme.md, can you fork project from my address and commit on readme-fa.md?
Number of readme.md lines is 1965, we can divide on two section, part one for me and part two for you,
Is it good for you?

**Comment by EmpireWorld** [comment]
@ghaseminya Yes. it is good plan. give me details. (which line i start translation?)

**Comment by ghaseminya** [comment]
I translate till line 1000, and you can start from line 1001 to end!

**Comment by EmpireWorld** [comment]
@ghaseminya Nice. :rocket: we started translation. :1st_place_medal:

**Comment by jwasham** [comment]
I meant to pass this along: https://github.com/jwasham/coding-interview-university/issues/125

**Comment by ghaseminya** [comment]
@jwasham ok, thanks

**Comment by jafarchoupan** [comment]
vvv

**Comment by mojtabaRKS** [comment]
hello @ghaseminya @EmpireWorld 
i couldn't see any translations in your repo !
where is your work ?
if you didn't any translation i would happy to do it !

**Comment by EmpireWorld** [comment]
> hello @ghaseminya @EmpireWorld
> i couldn't see any translations in your repo !
> where is your work ?
> if you didn't any translation i would happy to do it !

Hi there, unfortunately, I didn't get any time to translate this project, so it would be awesome if you're willing to do so.

Enjoy creating, stay awesome.

**Comment by develhopper** [comment]
Hello, since no one has started translating, so I will take over and let you know.

**Comment by jwasham** [comment]
I'm adding a link to this issue for translation in progress.

Note there is a translation file here:
https://github.com/jwasham/coding-interview-university/blob/main/translations/README-fa.md

**Comment by hamed-rv** [comment]
Hi, 

@jwasham Is the translation still in progress? I saw the README-fa.md file, there are lots of things needed to be translated. I can follow up on this ticket and complete the translation.

**Comment by jwasham** [comment]
Please take over. I don't see any pull requests.

**Comment by hamed-rv** [comment]
Sure, thanks
